using_the_api.md

```diff
@@ -0,0 +1,344 @@
+# The DATA Act Data Store Application Programming Interface (API)
```

Comment:
Let's just call this The USAspending API since nobody outside of this team knows what the data store is :)